prime/src/components/borrow/uniswap/UniswapPositionsTable.tsx

```diff
@@ -237,11 +237,16 @@ export default function UniswapPositionTable(props: UniswapPositionsTableProps)
 
   // This just saves us from having to specify all the args every time we use it.
   const formatPriceRatio_ = (p: GN) => {
-    formatPriceRatioGN(p, marginAccount.token0.decimals, marginAccount.token1.decimals, isInTermsOfToken0);
+    return formatPriceRatioGN(p, marginAccount.token0.decimals, marginAccount.token1.decimals, isInTermsOfToken0);
   };
 
-  const rows = uniswapPositionInfo.map((item: UniswapPositionInfo) => {
+  let rows: Array<JSX.Element[]> = [];
+  for (const item of uniswapPositionInfo) {
+    // TODO: sometimes we have issues with undefined values here, investigate
```

Comment:
I don't really like leaving this as a TODO

Reply:
Don't worry. It will be removed in the next PR. I put this here for me to remember to double-check this.